src/App.js

```diff
@@ -5,7 +5,16 @@ function App() {
   return (
     <div>
       <Header />
-      <Card checked={false} />
+      <Card
+        caption="Таблица цветов"
+        text="Для сохранения единообразия страниц, цвета в МедиаВики нужно
+        использовать обдуманно. Яркие цвета в статьях должны использоваться
+        только в виде исключения, когда это целесообразно по содержанию (на
+        своей странице участника можете брызгать краской по своему вкусу :-). В
+        порталах допускается более интенсивная раскраска, но ориентируйтесь на
+        существующие примеры."
+        checked={false}
+      />
```

Comment:
Could you please create array-of-objects variable and place it into the App state?
And could you also create a list of cards, that will be filled in the loop from that array-of-objects state?